akka-bbb-apps/src/main/scala/org/bigbluebutton/Boot.scala

```diff
@@ -115,4 +119,19 @@ object Boot extends App with SystemConfiguration {
   )
 
   val bindingFuture = Http().bindAndHandle(apiService.routes, httpHost, httpPort)
+
+  new GrpcServer(system, bbbActor).run()
 }
+
+class GrpcServer(system: ActorSystem, bbbActor: ActorRef) {
+  def run(): Future[Http.ServerBinding] = {
+    implicit val sys: ActorSystem = system
+    implicit val ec: ExecutionContext = sys.dispatcher
+    implicit val bbb: ActorRef = bbbActor
+
+    val service: HttpRequest => Future[HttpResponse] = MeetingServiceHandler(new MeetingServiceImpl())
+    val binding = Http().newServerAt("127.0.0.1", 8081).bind(service)

```

Comment:
This values should be configurable.

Reply:
Should both the host and the port be configurable or just the port?

Reply:
both, just like redis and postgres!